root/admin/attributes/Index.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * @flow
+ * Copyright (C) 2019 Anirudh Jain 
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import Layout from '../../layout';
+
+type attributeT = {
+  id: number,
+  name: string,
+  description: string,
+  childOrder: number,
+  parentID: number,
+  year: number,
+  hasDiscids: string,
+  entityType: string,
+  freeText: string
+}
+
+type Props = {
+  attributes: Array<attributeT> | null,
+  model: string,
+  models: Array<string>,
```

Comment:
Clearly models can either be undefined or null, otherwise "models ? this : that" on line 35 would be pointless :)

Reply:
It gave me flow errors when I allowed it to be null, let me see what I can do

Reply:
the type of model cant be null but models can be so I am changing just that.